src/main/webapp/js/user-page-loader.js

```diff
@@ -70,8 +70,10 @@ function fetchMessages() {
 
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
-  addLinks(); 
+  addLinks();
   setPageTitle();
-  showMessageFormIfLoggedIn()
+  showMessageFormIfLoggedIn();
   fetchMessages();
+  const config = { removePlugins: ['ImageUpload', 'Table', 'MediaEmbed'] };
```

Comment:
Let's remember to look back into this if we decide to implement image upload. How did you decide which plugins to remove?

Reply:
These three were not working when I tested them so I'm assuming they need more behind the scenes implementations to work so I removed them at least for now so we don't have any options that don't work

Reply:
Sounds good! :+1: